_docs/10_general/yourprivacypolicy.md

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
Don't forget to add the front matter, it's better to copy paste another doc first, so you have the required fields.

```
---
title: Your GDPR
category: general
category_order: 1
order: 10
permalink: /gdpr
---
```